_directory/data/joachimdoerr/data.yml

```diff
@@ -0,0 +1,29 @@
+---
+# name (required)
+# may be your real name or your nick
+name: Joachim Dörr
+
+# geo location (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+# you may use services like openstreetmap.org to find out your geo location
+latitude: 49.352560
+longitude: 7.254550
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
+# should be 300 chars at maximum. no markup or line breaks!
+bio: "Hello, I am an active Friends Of REDAXO member and working on my leaded FOR AddOns in my freetime. I’m involved in REDAXO since version 3."
```

Comment:
I am no native speaker at all.. „my leaded addons“ sounds strange, I guess it needs to be worded differently

Reply:
Yeah, it’s not clearly understandable. And because they’re very popular, you should mention MForm and MBlock, @joachimdoerr? So how about something like this:

> Hi, I'm an active member of Friends Of REDAXO and I'm working on FOR add-ons in my spare time, especially the popular MForm and MBlock. I am a member of REDAXO since version 3.

Reply:
I think your version @schuer is well.